content/presubmission_inquiry.md

```diff
@@ -38,11 +37,11 @@ inquiry, the following rules have been proposed:
 1. Lorem ipsum
 1. Lorem ipsum
 
-These rules are chosen so that readers of varying experience levels will be able to gain from the article and range from the high level (Should I use deep learning for this problem?) to implementation best practices.
+These rules have been devised to effectively reach audiences of varying DL expertise readers ranging from high level guidance to implementation of best practices.
```

Comment:
One typo, but I think the phrasing is a bit awk. Would revise to: "These rules range from high-level guidance to implementation of best practices and they have have been devised to effectively reach audiences of varying expertise"